CONTRIBUTING.md

```diff
@@ -52,10 +52,10 @@ sight.
 3. List references in a remote repository: `git ls-remote --refs origin`
 4. Open [Pull Request (PR) page](https://github.com/nabaztag2018/pynab/pulls) and memorize the PR number you want to test
 5. Switch to the PR code, here PR #123, change number for your test: `git fetch origin pull/123/head:pr/123 && git checkout pr/123`
```

Comment:
Should rather be:
`git fetch origin master && git fetch origin pull/123/head:pr/123 && git checkout pr/123 && git merge master`
to ensure having **current** state of master branch **plus** commits from given PR.